charts/node/templates/_helpers.tpl

```diff
@@ -88,9 +88,16 @@ Create the name of the service account to use
 Create the database path depending on the database backend in use (rocksdb or paritydb)
 */}}
 {{- define "node.databasePath" -}}
-{{- if eq .Values.node.database "paritydb" }}
+{{- if eq .Values.node.chainData.database "paritydb" }}
 {{- "paritydb" }}
 {{- else }}
 {{- "db" }}
 {{- end }}
 {{- end }}
+
+{{/*
+Define a regex matcher to check if the passed node flags passed are already managed by the chart
+*/}}
+{{- define "node.chartManagedFlagsRegex" -}}
+{{- "(\\W|^)(--name|--base-path|--chain|--validator|--collator|--light|--database|--pruning|--prometheus-external|--prometheus-port|--node-key|--wasm-runtime-overrides|--jaeger-agent|--rpc-external|--unsafe-rpc-external|--ws-external|--unsafe-ws-external|--rpc-methods|--rpc-cors|--rpc-port|--ws-port)(\\W|$)" }}
```

Comment:
Maybe move this to _helpers.tpl ?

Reply:
It is already in `charts/node/templates/_helpers.tpl`